resources/models/reservation.py

```diff
@@ -341,6 +351,21 @@ def _save_dt(self, attr, dt):
     def _get_dt(self, attr, tz):
         return get_dt(self, attr, tz)
 
+    @cached_property
+    def invoice_status(self):
+        status = None
+        if self.invoice_requested and hasattr(self, "order"):
+            invoice = getattr(self.order, "invoice")
+            status = ReservationInvoiceStatus.TO_BE_MARKED_AS_READY
+            # TODO: Handle the case there's been an error
```

Comment:
Is this TODO going to be handled now or later?

Reply:
Later, if needed. Currently `Invoice.sent_to_sap_at `is not set if there's an error in sending it to the SFTP server. That should be sufficient for us.